app/controllers/users_controller.rb

```diff
@@ -41,9 +41,11 @@ def create
     if @user.persisted?
       @user.tune_after_persisted(current_organization)
       if params[:more]
-        redirect_to new_user_path, notice: I18n.t("users.new.user_created_add")
+        redirect_to new_user_path, notice: I18n.t("users.new.user_created_add",\
+        uid:@user.member(current_organization).member_uid, name: @user.username)
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Space missing after colon.